server/pkg/explorer/client.go

```diff
@@ -0,0 +1,68 @@
+package explorer
+
+import (
+	proxy "github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/client"
+	proxyTypes "github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/types"
+)
+
+var (
+	endpoints = map[string]string{
+		"main": "https://gridproxy.grid.tf",
+		"test": "https://gridproxy.test.grid.tf",
+		"qa":   "https://gridproxy.qa.grid.tf",
+		"dev":  "https://gridproxy.dev.grid.tf",
+	}
+)
+
+type ExplorerClient struct {
+	proxyClient proxy.Client
+}
+
+func NewClient() *ExplorerClient {
+	return &ExplorerClient{}
+}
+
+func (c *ExplorerClient) Load(net string) error {
+	endpoint := endpoints["dev"]
+	if res, ok := endpoints[net]; ok {
+		endpoint = res
+	}
+	c.proxyClient = proxy.NewClient(endpoint)
+	return nil
+}
+
+func (c *ExplorerClient) Ping() error {
+	return c.proxyClient.Ping()
+}
+
+func (c *ExplorerClient) Nodes(params NodesRequestParams) ([]proxyTypes.Node, error) {
+	nodes, _, err := c.proxyClient.Nodes(params.Filters, params.Pagination)
```

Comment:
what is the second argument returned here?

Reply:
To do proper pagination, a client would definetly need the `total_count`. Don't you agree?

Reply:
yes my bad. i thought the returned count is the length of the returned list. but it is the total count of all matched nodes on the grid. edited to return the count as well